docs/HowTo/Use/Revert-Reason.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Including revert reason in transactions with GoQuorum
+---
+
+# Revert reason
+
+In smart contracts, the
+[`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) operation
+triggers an exception to flag an error and revert the current call. The EVM passes back to the
+client an optional string message containing information about the error.
+
+!!! example
+
+    ```sol
+    pragma solidity >=0.5.0 <0.7.0;
+
+    contract VendingMachine {
+        function buy(uint amount) public payable {
+            if (amount > msg.value / 2 ether)
+                revert("Not enough Ether provided");
+            // Alternative way to do it:
+            require(
+                amount <= msg.value / 2 ether,
+                "Not enough Ether provided."
+            );
+            // Perform the purchase.
+        }
+    }
+    ```
+
+## Enabling revert reason
+
+Use the [`--revertreason`](../../Reference/CLI-Syntax.md#revertreason)
+command line option to include the revert reason in the transaction receipt.
+
+## Where is the revert reason included
+
+With revert reason enabled, the transaction receipt returned by
+[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_getTransactionReceipt) includes
```

Comment:
```suggestion
[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_gettransactionreceipt) includes
```

Sorry we didn't catch this before, but headers in URLs need to be lowercase to work.